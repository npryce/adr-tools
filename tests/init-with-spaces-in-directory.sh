adr init "test doc/adr"
