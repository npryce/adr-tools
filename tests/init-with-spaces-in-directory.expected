adr init "test doc/adr"
test doc/adr/0001-record-architecture-decisions.md
